esx_service/vmdk_ops.py

```diff
@@ -500,9 +500,13 @@ def get_vol_path(datastore, tenant_name=None):
 
 
 def known_datastores():
-    """returns names of know datastores"""
+    """returns names of known datastores"""
     return [i[0] for i in vmdk_utils.get_datastores()]
 
+def get_latest_datastores():
```

Comment:
Yes, I was thinking on same line. However, handling datastore cache misses in `parse_vol_name` seems weird. So, proposed to change "get_list & validate" to "validate".